pkg/transforms/http.go

```diff
@@ -187,6 +188,12 @@ func (sender *HTTPSender) httpSend(ctx interfaces.AppFunctionContext, data inter
 
 	req.Header.Set("Content-Type", sender.mimeType)
 
+	// Set all the http request headers
+	for key, element := range sender.httpRequestHeaders {
+		req.Header.Set(key, element)
+
```

Comment:
Need to add unit test for this. 

Will be similar to `TestHTTPPostPutWithSecrets` where it checks the header for the expected elements.https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/pkg/transforms/http_test.go#L174-L179